sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/KeepLegacyOutputs.scala

```diff
@@ -36,6 +36,14 @@ object KeepLegacyOutputs extends Rule[LogicalPlan] {
           assert(s.output.length == 3)
           val newOutput = s.output.head.withName("database") +: s.output.tail
           s.copy(output = newOutput)
+        case s: ShowTableExtended =>
```

Comment:
I think this change should come after `ShowTableExtended` being the default, with tests to make sure v1/v2 commands have the same behavior.

Reply:
Shall we implement `ShowTableExtendedExec` for v2 command `ShowTableExtended` or support it in existing `ShowTableExec` like v1 `ShowTableCommand`?